src/android/FirebasePluginMessagingService.java

```diff
@@ -78,7 +78,7 @@ public void onMessageReceived(RemoteMessage remoteMessage) {
             title = remoteMessage.getNotification().getTitle();
             text = remoteMessage.getNotification().getBody();
             id = remoteMessage.getMessageId();
-        } else {
```

Comment:
shouldn't this be an else/if?  If not, please fix the formatting so that the new if statement is on a different line.